app/controllers/FinancialDetailChargesController.scala

```diff
@@ -43,6 +43,9 @@ class FinancialDetailChargesController @Inject()(authentication: AuthenticationP
         case Right(chargeDetails) =>
           logger.debug("[FinancialDetailChargesController][getChargeDetails] - Successful Response: " + chargeDetails)
           Ok(Json.toJson(chargeDetails))
+        case Left(error: UnexpectedChargeResponse) if error.code == 404 =>
+          logger.info("[FinancialDetailChargesController][getChargeDetails] - 404 error: " + error)
+          Status(error.code)(error.response)
         case Left(error: UnexpectedChargeResponse) if error.code >= 400 && error.code < 500 =>
```

Comment:
shouldFix: status >= 400 remove equal to sign

Reply:
Please use Http.Status codes instead of "magic numbers"
like: https://www.playframework.com/documentation/2.0.2/api/java/play/mvc/Http.Status.html